### PR TITLE
Fixing issue of inconsistent QR code validation when adding and uploading through CSV file

### PR DESCRIPTION
**Pull request by shubham72a**

### CODE REVIEW

File: content-api/content-service/conf/application.conf [hunk 1]
```diff
@@ -748,7 +748,7 @@ collection {
 }
 
 #Request timeout
```
Comment (yogeshsarmalkar-trigyn) [inline content-api/content-service/conf/application.conf:750]: Revert this change. not required for this ticket

**Review by yogeshsarmalkar-trigyn — APPROVED** [review]

**Review by samarthtrigyn — APPROVED** [review]
